src/components/view/Contact/ContactNotesCard/index.js

```diff
@@ -0,0 +1,31 @@
+
+import React, { Component } from 'react';
+
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+class ContactProfileCard extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {userInfo} = this.props;
+    return (
+      <div>
+        <Card initiallyExpanded={true} >
+          <CardHeader
+              title='Notes'
+              titleColor={this.props.titleColor}
+              actAsExpander={true}
+              showExpandableButton={true}
+          />
+          <CardText expandable={true}>
+            notes props here
```

Comment:
This text should come from the contact's info, through `props`, right?

Reply:
ultimately, yeah it should